uav_cloud: /data/digiforest/steim_am_rhein/2023-08-15-10-24-10-exp04/prefor_steim_am_rhein_normals_exp04_map.ply
mls_cloud_folder: /data/digiforest/steim_am_rhein/2023-08-15-10-24-10-exp04/payload_clouds_in_map
offset: [0.0,0.0,0.0]
output_folder: /data/digiforest/steim_am_rhein/2023-08-15-10-24-10-exp04/transformed_clouds_payloads_in_map_bis
crop_mls_cloud: True
mls_feature_extraction_method: "tree_segmentation"
debug: False
pose_graph_file: /data/digiforest/steim_am_rhein/2023-08-15-10-24-10-exp04/slam_pose_graph.g2o
save_pose_graph: True
icp_fitness_score_threshold: 0.75
